pkg/volume/plugins.go

```diff
@@ -33,14 +33,21 @@ import (
 )
 
 // VolumeOptions contains option information about a volume.
-//
-// Currently, this struct containers only a single field for the
-// rootcontext of the volume.  This is a temporary measure in order
-// to set the rootContext of tmpfs mounts correctly; it will be replaced
-// and expanded on by future SecurityContext work.
 type VolumeOptions struct {
 	// The rootcontext to use when performing mounts for a volume.
+	// This is a temporary measure in order to set the rootContext of tmpfs mounts correctly.
+	// it will be replaced and expanded on by future SecurityContext work.
 	RootContext string
+
+	// The attributes below are required by volume.Creater
```

Comment:
Shouldn't this be `Creator`?  I hate to pick a nit at the last minute... follow-up if you agree?